bip-0105.mediawiki

```diff
@@ -62,10 +62,11 @@ Each time a miner creates a block, they may vote to increase or decrease the
 blocksize by a maximum of 10% of the current block size limit. These votes will 
 be used to recalculate the new block size limit every 2016 blocks.
 
-Votes are cast using the block's coinbase field.
+Votes are cast using the block's coinbase transaction scriptSig.
 
-The first 4 bytes of the coinbase field shall be repurposed for voting as an
-unsigned long integer which will be the block size in bytes.
+As per BIP34, the coinbase transaction scriptSig starts with a serialized 
+CScript "<< nHeight". The the preferred block size will follow as a push of a 
+integer in bytes "<< nSizeVote". For example, 0x4c(OP_PUSHDATA1) 0x03(size of constant) 0x80841e(2MB).
```

Comment:
Probably should stick to bytes here to be clear the 2 MB value is signed LE? And you need 4 bytes for this to account for the sign.